var/spack/repos/builtin/packages/gunrock/package.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Gunrock(CMakePackage, CudaPackage):
+    """High-Performance Graph Primitives on GPUs"""
+
+    homepage = "https://gunrock.github.io/docs/"
+    url      = "https://github.com/gunrock/gunrock/archive/v1.1.tar.gz"
+    git      = "https://github.com/gunrock/gunrock.git"
+
+    version('develop', git=git, submodules=True)
```

Comment:
```suggestion
    version('master', submodules=True)
```
If you use `submodules`, it must be a git download, and you have git specified at the top, so this should work.